packages/dev/core/src/ShadersWGSL/background.fragment.fx

```diff
@@ -150,8 +150,8 @@ varying vPositionW: vec3f;
     }
 
     fn project(viewDirectionW: vec3f, eyePosition: vec3f) -> vec3f {
-        var radius: f32 = projectedGroundInfos.x;
-        var height: f32 = projectedGroundInfos.y;
+        var radius: f32 = uniforms.projectedGroundInfos.x;
+        var height: f32 = uniforms.projectedGroundInfos.y;

```

Comment:
These could also be `let`. I think we could use `let` instead of `var` in a lot of places... I'm asking the Matrix channel if it would be worth it...

Reply:
So, it seems that it wouldn't really be more efficient to use `let` instead of `var`, so let's ignore it.